saw-remote-api/src/SAWServer/LLVMCrucibleSetup.hs

```diff
@@ -77,68 +70,82 @@ instance FromJSON StartLLVMCrucibleSetupParams where
 
 data ServerSetupVal = Val (CMS.AllLLVM MS.SetupValue)
 
--- TODO: this is an extra layer of indirection that could be collapsed, but is easy to implement for now.
 compileLLVMContract ::
   (FilePath -> IO ByteString) ->
   BuiltinContext ->
+  (Map ServerName CMS.GhostGlobal) ->
```

Comment:
The parentheses around `Map ServerName CMS.GhostGlobal` are redundant.

Reply:
Fixed.